NEWS.md

```diff
@@ -1,60 +1,55 @@
-3.23.1
+3.24.1
 ======
 
+## Highlights
 
-## Features
-
- * The `redis()` destination now handles any number of command parameters.
-   (#2816)
-
- * The `format()` option of `date-parser()` supports lists. From now on, a
-   single `date-parser()` instance is able to process different date formats,
-   making it easy to catch on when some programs change the way they log
-   information. (#2779)
+ * Add a new template function called `format-flat-json()`, which generates
+   flattened json output. This is useful for destinations, where the json
+   parsing does not handle nested json format. (#2890)
+ * Add ISO 8601 compliant week numbering. Use it with the `${ISOWEEK}` macro
+   and the timestamp prefixes `P_, S_, R_, C_` (#2878)
+ * Add `add-contextual-data()` glob selector. It matches the message with shell
+   style globbing. Enable it by setting `selector(glob("$my_template")` in the
+   `add-contextual-data()` block. (#2936)
+ * Send Server Name Identification (SNI) information with `transport(tls)`.
+   Enable it by setting the `sli(yes)` option in the `tls` block in your
+   `destination`. (#2930)
 
- * Add relocation support for `disk-buffer()`. The `relocate` subcommand of
-   `dqtool` can be used to move a single or multiple queue files. (#2855)
-
- * `file(), pipe()`: The `time-reap()` option now can be set or disabled for
-   each destination separately.
-   Use `time-reap(0)` to disable closing idle destination files. (#2798)
+## Features
 
- * `syslog-ng-ctl`: Add `list-files` subcommand to print files present in the
-   current configuration. (#2797)
+ * Timezone related `rewrite` rules (#2818)
```

Comment:
New rewrite operations to manipulate the timezone portion of timestamps have been added. set-timezone() to set the timezone value to a specific value, fix-timezone() to fix up an incorrectly recognized timezone and guess-timezone() to automatically deduce the timezone value on the assumption that the message is received in near real time.

Reply:
Thanks fixed! :)